raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Range of remedies to the injured 
+(226) In order to enforce environmental standards and duties, the range of remedies available to a person injured or threatened to be injured shall include: [a] temporary or permanent injunctions; [b] specific performance orders; [c] fines and penalties; [d] compensation; [e] restoration orders, conservation orders, easement orders, and [f] compliance orders; [g] imprisonment; and [h] any other remedy as the Tribunal or court may determine or any administrative measure as may be prescribed by the Minister.
+
+Environmental performance bond 
+(227) [1] The Minister shall prescribe in the Regulations, activities mental or processes which threaten the environment of which environmental performance bond may be required. [2] An environmental performance bond shall be deposited with the Director of Environment as security for good environmental practice until its refund to the depositor. [3] The environmental performance bond shall be returned to the operator of activity or process upon the satisfaction of the conditions set by the Minister. [4] The Director of Environment shall give the operator of an activity or process an opportunity to be heard before confiscating the environmental performance bond. [5] An environmental performance bond shall be confiscated in whole or in part when the Director of Environment finds an operator's practices violate the provisions of this Act, including the conditions of any certificate, licence or permit issued under this Act. [6] Where an environmental performance bond is confiscated under subsection [5], it shall be used to rehabilitate a degraded environment. 
+
+Right to compensation 
+(228) [1] For the avoidance of doubt, the Minister shall make regulations prescribing the right to compensation for those who suffer damages while protecting the environment. [2] Any person who violates environmental protection standards causing any damage shall compensate for the damages and costs of remedying the consequences. 
+
+Environmental emergency preparedness 
+(229) [1] The Minister shall prepare guidelines for the management mental of environmental emergencies including: emergency preparedness [a] major oil spills and gas leakages; [b] spills of other hazardous substances; [c] industrial accidents; [d] natural and climate change related disaster such as floods, cyclones, droughts and major pest infestations or other intrusions of alien species of fauna and flora; [e] influx of refugees, and [f] fire. [2] With a view to prepare an emergency preparedness plan appropriate to the risk anticipated in any establishment, premises or any area of land, the Minister shall make consultation with the Disaster Management Department, Government and private institutions, relevant organization as well as individual persons. 
```

Comment:
Please replace:
* `management mental of` with `management of`;
* `emergency preparedness [a]` with `[a]`; and
* `refugees, and [f]` with `refugees; and [f]`.